graphistry/umap_utils.py

```diff
@@ -865,8 +866,7 @@ def umap(
         if dbscan:
             res = res.dbscan(min_dist=min_dist, kind=kind, fit_umap_embedding=True)  # type: ignore
 
-        if not inplace:
-            return res
+        return res
```

Comment:
Hmm most seemed just refactor but this may be a logic change ?

Reply:
reverted and added the overloads